lib/oofatfs/ff.c

```diff
@@ -2821,7 +2821,7 @@ static FRESULT create_name (    /* FR_OK: successful, FR_INVALID_NAME: could not
         if (di >= FF_MAX_LFN) return FR_INVALID_NAME;   /* Reject too long name */
         lfn[di++] = wc;                 /* Store the Unicode character */
     }
-    while (*p == '/' || *p == '\\') p++;    /* Skip duplicated separators if exist */
+    if(wc == '/' || wc == '\\') while (*p == '/' || *p == '\\') p++;    /* Skip duplicated separators if exist */
```

Comment:
Can you please put a single space between the `if` and the `(`.

Also, please reword the title of the commit message for this commit to:
```
lib/oofatfs: Fix speculative read in create_name.
```

Reply:
All fixed and pushed.